.github/workflows/ci.yml

```diff
@@ -4,9 +4,14 @@ on:
   push:
     branches: [master]
   pull_request:
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
 
 jobs:
   ci:
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
```

Comment:
We don't have to skip workflows usually, what's the use case here you imagine? This just complicates the config otherwise.

Reply:
@JohnTitor when you just edit README.md without code commit message can be `[ci skip] update README`

Reply:
It's harmless as I mentioned earlier:
> We usually don't have to run workflow manually and don't have to skip workflow as this public project doesn't have a GHA hours limit (the concurrency limit is also not an issue here) and the number of submitted PRs is quite small.